src/backend/providers/android_apps/AndroidAppsMetadata.cpp

```diff
@@ -101,9 +101,9 @@ MetadataHelper::MetadataHelper(QString log_tag)
     , rx_meta_itemprops(QStringLiteral(R""(<meta itemprop="(.+?)" content="(.+?)")""), QRegularExpression::DotMatchesEverythingOption)
     , rx_background(QStringLiteral(R""(<meta property="og:image" content="(.+?)")""))
     , rx_developer(QStringLiteral(R""(<a href="/store\/apps\/dev(eloper)?\?id=.+?" class=".*?">([^<]+)<\/a><\/span>)""))
-    , rx_category(QStringLiteral(R""(<a itemprop="genre".*?>([^<]+)<\/a>)""))
+    , rx_category(QStringLiteral(R""(href="\/store\/apps\/category\/([^"]+)" aria-label)""))
     , rx_screenshots(QStringLiteral(R""(<img src="([^"]+=w720-h310)")""))
-    , rx_release(QStringLiteral(R""("Google Commerce Ltd"],"(.+?)")""))
+    , rx_release(QStringLiteral(R""("\]\]\],\["(.+?[0-9]{4})",\[)""))
```

Comment:
While the whole code is sensitive to site changes, this solution seems especially so. Let's try to look for the string `Updated on` on the page.

Reply:
I also had an alternative solution using the `Updated on` string, but it was not working for me. Could you check this modification below?

```suggestion
    , rx_release(QStringLiteral(R""(Updated on<\/div><div class="[^"]+">(.+?)<\/)""))
```

Reply:
I won't be able to check this the weekend, but I can take a look next week.

Reply:
Sorry about the late reply, but I can now confirm that the `Updated on` solution works for me.